thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/alert/scheme/DetectionEmailAlerter.java

```diff
@@ -62,7 +63,10 @@
 
   private static final Comparator<AnomalyResult> COMPARATOR_DESC =
       (o1, o2) -> -1 * Long.compare(o1.getStartTime(), o2.getStartTime());
-  private static final String EMAIL_WHITELIST_KEY = "emailWhitelist";
+
+  private static final String PROP_EMAIL_WHITELIST = "emailWhitelist";
+  private static final String PROP_ADMIN_RECIPIENTS = "adminRecipients";
```

Comment:
Who are admin recipients?
Is it per subscription group or per application?
What is the different between "adminReceipients" and "cc" list?

Reply:
Well, adminRecipients are not tied with either an application or subscription group. By admin, I am referring to the platform admins (us). We need to be part of all the alerts that are sent out.

For example, we include thirdeye-dev alias in cc for all the email alerts we send out. This will give us insights into which alerts are very actively monitored (email threads), which conversations ended up in critical GCNs and we can also help them with RCA.

Reply:
Sure that totally makes sense. Then is it configured in Yaml?

Reply:
Yes, this will be configured in the ThirdEye application config - detector.yml